Assets/MixedRealityToolkit.Examples/Experimental/PulseShader/Scenes/PulseShaderExamples.unity

```diff
@@ -233,7 +233,7 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 83d9acc7968244a8886f3af591305bcb, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
-  activeProfile: {fileID: 11400000, guid: 7e7c962b9eb9dfa44993d5b2f2576752, type: 2}
+  activeProfile: {fileID: 11400000, guid: 31a611a779d3499e8e35f1a2018ca841, type: 2}
```

Comment:
Question does this shader leave the meshes visible after the pulse? If not, i would recommend _not_ changing the default material.

Reply:
On further consideration, we should not include experimental components in the default profiles.

Thanks!

Reply:
Reverted DefaultMixedRealitySpatialAwarenessMeshObserverProfile. Thanks!